README.md

```diff
@@ -4,35 +4,90 @@ NodeJS >=18.19.0
+├── svelte.config.js // Svelte(kit) configuration - contains the configurations for crawling and SSG as well
+├── update-cover-url-overwrites.sh // See "Utility Scripts" for info
+```
+
+## Project Routing
+
+The routes are determined by file path starting at `src/routes`. For example, (`src/routes/community-hub/+page.svelte`)[src/routes/community-hub/+page.svelte] renders under (https://bsaber.com/community-hub)[https://bsaber.com/community-hub] or in the local development environment under (http://localhost:3000/community-hub)[http://localhost:3000/community-hub].
+
+Note the following when it comes to those:
+
+1. **We Statically Generate our Site**: You cannot rely on server side routing to dynamically update the pages. These only execute once when rendering and generating the static HTML and CSS files and cannot react to, for example, a change in the file system. Creating and managing files will thus also not usually make sense.
+
+2. **Pages with Path Parameters Need to be Linked**: Paths that contain path parameters such as `[page]` need to be linked to somehwere from another page that either has no path parameter or is linked by a page fulfilling these conditions. Basically, they need to be crawlable by following links. If a page cannot be crawled starting from any non-parameter path it will simply exist in the final product.
+
+3. **`npm run dev` =/= Production Environment**: You cannot rely on it 100%. For example, non-rendered pages will still work with `npm run dev` since it actually executes server-side.
+
+4. **You usually want to stick to `+page.server.ts` not `+page.ts`**: The difference between the two is, that the `.server.ts` only executes server side (so once when rendering). The `+page.ts` will run server and client side. Due to the way we use server-side code it usually doesn't make sense to use `+page.ts` and can quickly result in confusing errors.
+
+5. **API functionality won't get you far**: Svelte allows you to define an API on the server side. This will also work only very limitedly. At the very list the dynamic nature of it will be stripped away.
 
 ## Contributing
 
-### Before submitting a PR
+We are available for question, support and discussion on our (Discord server)[https://discord.gg/VJZHUbt].
 
-Verify that the app can be built:
+We would hate for you to spend valuable time on something that we end up not using. Here are some infos on how to increase the likelihood of us merging your changes:
 
-```
-npm run build
-```
+- You can see the tasks that we decided to work and which are not worked on by anyone else as (approved, but non-assigned Issues)[https://github.com/beastsaber/bsaber/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved+no%3Aassignee]. Do watch the labels!
 
-Run prettier formatter:
+- If you are unsure if you can work on a task or what it entails: Do contact us. Either comment on the issue or drop by on the Discord. We are for most questions and any doubt on your part!
```

Comment:
The last sentence is missing a word, could be: "We are happy to answer your questions and concerns."